### PR TITLE
Fix: toggle styles

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/v2/Toggle/styles.ts [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
+
 export const useStyles = () => {
   const theme = useTheme();
   return css`
-    width: 44px;
-    height: 22px;
+    width: calc(${thumbSize} * 2);
+    height: ${thumbSize};
     padding: 0;
     & .MuiSwitch-switchBase {
```
Comment (therealemjy) [inline src/components/v2/Toggle/styles.ts:12]:
```suggestion
    &.MuiSwitch-switchBase {
```

**Review by therealemjy — APPROVED** [review]

File: src/components/v2/Toggle/styles.ts (hunk 1 above)
Comment (baomastr) [reply src/components/v2/Toggle/styles.ts:12]: No, the space has been deliberately removed here in order to increase the specificity of the styles.

Comment (baomastr) [reply src/components/v2/Toggle/styles.ts:12]: Well, I also want to note that such class inheritance hurts more than the use of native optimization, which mui provides and is also completely incapacitated in the context of other libraries.

Comment (therealemjy) [reply src/components/v2/Toggle/styles.ts:12]: You mean the space has been added? Maybe I understand this wrong, but then in this case you can just get rid of the `&`.

Comment (therealemjy) [reply src/components/v2/Toggle/styles.ts:12]: By the way this is minor, feel free to merge

**Pull request merged**